joomla/joomla-cms
# Fixed broken view in administrator/components/com_languages/views/installed/tmpl/default.php under smaller resolutions

New issue:
Broken view in administrator/components/com_languages/views/installed/tmpl/default.php under smaller resolutions. The `Native Title` header ins't removed in smaller resolutions due to missing classes

### Summary of Changes
Details:
- Fixed `Native Title` header
- Also centered the  `Language Tag` column

### Testing Instructions
1) Test admin interface installed languages without applying patch. Try different resolutions and notice that in smaller (tablet & phone) resolutions, headers and data columns are misaligned.
2) Apply patch and test again. Headers and columns should now be aligned

### Documentation Changes Required
None
